packages/store/src/internal/config/ngxs-config.ts

```diff
@@ -0,0 +1,48 @@
+import { Injectable, Type } from '@angular/core';
+
+import { ObjectKeyMap } from '../internals';
+import { NgxsCompatibility } from '../../symbols';
+import { NgxsExecutionStrategy } from '../../execution/symbols';
+import { DispatchOutsideZoneNgxsExecutionStrategy } from '../../execution/dispatchOutsideZoneNgxsExecutionStrategy';
+
+/**
+ * The NGXS config settings.
+ */
+@Injectable()
+export class NgxsConfig {
+  /**
+   * Run in development mode. This will add additional debugging features:
+   * - Object.freeze on the state and actions to guarantee immutability
+   * (default: false)
+   */
+  developmentMode: boolean;
+
+  /**
+   * Support a strict Content Security Policy.
+   * This will circumvent some optimisations that violate a strict CSP through the use of `new Function(...)`.
+   * (default: { strictContentSecurityPolicy: false })
+   */
```

Comment:
This description is now against the wrong thing, it used to be on the `strictContentSecurityPolicy` property inside the compatibility options. Please move this interface definition to be inline again and move the defaults back to the constructor.

Reply:
I do not agree with the fact that we have to transfer everything to the constructor, since this is an unnecessary gesture, we already specify the default value in advance, and here it is immediately obvious!